docs/thick-plugin.md

```diff
@@ -53,18 +53,6 @@ kubectl apply -f deployments/multus-daemonset-thick.yml
 
 ### Command line parameters
 
-Multus thick plugin variant accepts the same
-[entrypoint arguments](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/how-to-use.md#entrypoint-script-parameters)
-its thin counterpart allows - with the following exceptions:
-
-- `additional-bin-dir`
-- `binDir`
-- `cleanup-config-on-exit`
-- `cniDir`
-- `multus-kubeconfig-file-host`
-- `rename-conf-file`
-- `skip-multus-binary-copy`
-
 It is important to refer that these are command line parameters to the golang
 binary; as such, they should be passed using a single dash ("-") e.g.
 `-additional-bin-dir=/opt/multus/bin`, `-multus-log-level=debug`, etc.
```

Comment:
`-additional-bin-dir` and `-multus-log-level` are no longer exist, so we may change.

For your info, now we only have `-config` and `-version`.
```
$ ./bin/multus-daemon --help
Usage of ./bin/multus-daemon:
  -config string
        Specify the path to the multus-daemon configuration (default "/etc/cni/net.d/multus.d/daemon-config.json")
  -version
        Show version
$
```

Reply:
Done.